src/org/jitsi/impl/neomedia/rtp/sendsidebandwidthestimation/SendSideBandwidthEstimation.java

```diff
@@ -66,6 +86,31 @@
      */
     private static final int kLimitNumPackets = 20;
 
+    /**
+     * send_side_bandwidth_estimation.cc
+     */
+    private static final float kDefaultLowLossThreshold = 0.02f;
+
+    /**
+     * send_side_bandwidth_estimation.cc
+     */
+    private static final float kDefaultHighLossThreshold = 0.1f;
+
+    /**
+     * send_side_bandwidth_estimation.cc
+     */
+    private static final int kDefaultBitrateThresholdKbps = 0;
+
+    /**
+     * Disable the loss experiment by default.
+     */
+    private static final float kDefaultLossExperimentPct = 0;
```

Comment:
Pct for "percent"? If I'm reading right it's used as a probability (0-1, not 0-100)

Reply:
Probability <> percent and in this particular case we care about the percentage, in some form. What this variable holds is the proportion of lost packets; I felt that kDefaultLossExperimentProportion would be too long and that percentage (abbreviated as Pct) is clear enough. I'll finally rename the variable to kDefaultLossExperimentProportion.